docs/api/security/lifecycle/psa_lifecycle.md

```diff
@@ -0,0 +1,28 @@
+## PSA lifecycle
+
+
+### Description
+
+PSA Lifecycle allows fine grained control over target RoT without compromising on developer experience.
```

Comment:
It is not completely clear to me what you mean by "allows fine-grained control over the target RoT". Can we state explicitly what you can control with respect to the RoT?